front_end/Matching/HomeScreen.js

```diff
@@ -1 +1,24 @@
-//Matching-Home
\ No newline at end of file
+//Matching-Home
+import React, {Component} from 'react';
+import {
+    SafeAreaView,
+} from 'react-native';
+
+import QuestionList from './QuestionList';
+
+export default class HomeScreen extends Component {
```

Comment:
I'm not sure what your structure looks like - but is this truly the app home screen? I suggest something like ```MatchingHomeScreen``` just to be more explicit